### PR TITLE
Consolidate DeepWork review rules

### PR DESCRIPTION
**Pull request by nhorton**

### CODE REVIEW

File: .deepwork/schemas/test_traceability_comments/deepschema.yml [hunk 1]
```diff
@@ -4,13 +4,34 @@ instructions: |
...
+  `describe(...)` block:
+
+  ```ts
+  // THIS TEST VALIDATES A HARD REQUIREMENT (BRIDL-REQ-008.3).
+  // YOU MUST NOT MODIFY THIS TEST UNLESS THE REQUIREMENT CHANGES.
+  it("validates durable behavior", () => {
+    // ...
+  });
+  ```
+
```
[the remark this answers is not in the document]
Comment (nhorton) [reply .deepwork/schemas/test_traceability_comments/deepschema.yml:20]: Addressed in 9b6156e by restoring the carve-out in the DeepSchema instructions and adding an explicit `no-false-positive-for-unclaimed-coverage` requirement so utility/edge-case tests without formal requirement coverage claims are not flagged.

File: .deepwork/schemas/test_traceability_comments/deepschema.yml [hunk 2]
```diff
@@ -4,13 +4,41 @@ instructions: |
...
   - "tests/**/*.ts"
   - "tests/**/*.tsx"
   - "**/*.test.ts"
   - "**/*.spec.ts"
 
 requirements:
+  required-two-line-traceability-comment: >
+    Tests that validate a formal requirement MUST include both traceability
+    comment lines: `THIS TEST VALIDATES A HARD REQUIREMENT (BRIDL-REQ-NNN.M).`
+    and `YOU MUST NOT MODIFY THIS TEST UNLESS THE REQUIREMENT CHANGES.`
```
[the remark this answers is not in the document]
Comment (nhorton) [reply .deepwork/schemas/test_traceability_comments/deepschema.yml]: Addressed in ff8159e by allowing the first traceability line to contain one or more requirement IDs, including comma-separated IDs such as `BRIDL-REQ-005.3, BRIDL-REQ-005.6`, so existing multi-requirement tests are not incorrectly flagged.

File: .deepreview [hunk 3]
```diff
@@ -223,7 +130,12 @@ doc_consistency:
     additional_context:
       unchanged_matching_files: true
     instructions: |
-      Review changed requirements and verify that the documentation stays current.
+      Review changed requirements and verify that repository documentation and
+      public documentation site content stay current. This rule provides
+      judgment-based review coverage for keeping docs aligned with formal
+      requirement changes, including adapter boundaries, controllable-element
+      documentation, tack/state persistence, and other product requirements
+      that affect public claims.
```
[the remark this answers is not in the document]
Comment (nhorton) [reply .deepreview]: Addressed in ff8159e by expanding `doc_consistency` match coverage to include the repository docs, doc_site files, and `src/agents/pi/PiAdapter.ts`, with `unchanged_matching_files: true`, so the consolidated rule receives the relevant documentation/implementation context.

File: doc/.deepreview [hunk 4]
```diff
@@ -0,0 +1,30 @@
+controllable_elements_matrix:
+  description: "Review controllable element mappings and support-status claims for judgment-based accuracy."
+  match:
+    include:
+      - "controllable-elements.md"
```
[the remark this answers is not in the document]
Comment (nhorton) [reply doc/.deepreview:5]: Leaving this matcher as-is: `.deepreview` files are scoped by directory (per the DeepReview schema guidance), so `doc/.deepreview` governs paths relative to `doc/`. This was verified by `deepwork_get_configured_reviews`, which discovers the rule at `doc/.deepreview` with include `controllable-elements.md`, and by `/review`, which ran the doc-scoped rules from this branch. Using `doc/controllable-elements.md` inside `doc/.deepreview` would incorrectly point at `doc/doc/controllable-elements.md` under the scoped semantics.

File: scripts/.deepreview [hunk 5]
```diff
@@ -0,0 +1,27 @@
+release_metadata_sync:
+  description: "Review Bridl release metadata synchronization automation."
+  match:
+    include:
+      - "sync-release-version.mjs"
```
[the remark this answers is not in the document]
Comment (nhorton) [reply scripts/.deepreview:5]: Leaving this matcher as-is for the same reason: this `scripts/.deepreview` file is scoped to the `scripts/` directory, so `sync-release-version.mjs` is the correct local match. `deepwork_get_configured_reviews` and `/review` both discovered and ran this rule on the branch.

File: .github/workflows/.deepreview [hunk 6]
```diff
@@ -0,0 +1,26 @@
+release_workflow_safety:
+  description: "Review Bridl npm release workflow safety."
+  match:
+    include:
+      - "release.yml"
```
[the remark this answers is not in the document]
Comment (nhorton) [reply .github/workflows/.deepreview:5]: Leaving this matcher as-is: `.github/workflows/.deepreview` is scoped to `.github/workflows/`, so `release.yml` is the correct local match. The branch review run discovered this file-specific rule successfully.

File: tests/fixtures/integration/.deepreview [hunk 7]
```diff
@@ -0,0 +1,37 @@
+integration_fixture_consistency:
+  description: "Review integration fixture README files and catalog entries for scenario accuracy, structure, and write-back expectations."
+  match:
+    include:
+      - "INTEGRATION_TEST_FIXTURES.md"
+      - "**/*"
```
[the remark this answers is not in the document]
Comment (nhorton) [reply tests/fixtures/integration/.deepreview:6]: Leaving these matchers as-is: `tests/fixtures/integration/.deepreview` is scoped to `tests/fixtures/integration/`, so `INTEGRATION_TEST_FIXTURES.md` and `**/*` are local patterns for that fixture tree. This was verified by `/review`, which ran `integration_fixture_consistency` against the catalog and local `.deepreview` file.

File: .deepreview [hunk 8]
```diff
@@ -7,6 +7,8 @@ prompt_best_practices:
       - ".claude/**/*.md"
       - ".deepwork/review/*.md"
       - ".deepwork/jobs/**/*.md"
+      - "**/.deepreview"
+      - ".deepwork/schemas/**/*.yml"
```
[the remark this answers is not in the document]
Comment (nhorton) [reply .deepreview:11]: Addressed in 8ffec48 by making `prompt_best_practices` file-type agnostic: the description now says prompt/instruction files, and the instructions now say to review the file’s natural-language instructions rather than a markdown file specifically.
